lib/Preview/ProviderV2.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+// Maintainer's note:
+// ---
+// This file is a copy-paste from the nextcloud/server repository, moved to the OCA namespace, because we need it, but it's in the private \OC\Preview namespace.
+// https://github.com/nextcloud/server/blob/64208b6d2273269cd51bb6e4a7aa939a144bc93c/lib/private/Preview/ProviderV2.php
+// ---
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Preview;
```

Comment:
Please move it into the `OCA\Epubviewer\Preview` namespace so that it doesn't risk conflicting with other apps.